packages/client/tests/functional/issues/11233/tests.ts

```diff
@@ -6,6 +6,7 @@ import testMatrix from './_matrix'
 declare let prisma: PrismaClient
 declare let Prisma: typeof PrismaNamespace
 
+// https://github.com/prisma/prisma/issues/11233
```

Comment:
Look like an accidental change, the issue behind the link is irrelevant.

Reply:
```suggestion
```

Reply:
Are you sure, the test is "should not throw when using Prisma.empty inside $executeRaw" and the issue "Cannot pass Prisma.empty to $executeRaw function"

Reply:
Sorry, my bad. The context got mixed up in my head while I was scrolling, I was looking at the wrong piece of code. Sorry again for the confusion!

Reply:
(this file literally has the issue number in its name 😅)